code/ui/blocks/src/components/ArgsTable/ArgJsDoc.tsx

```diff
@@ -81,6 +88,11 @@ export const ArgJsDoc: FC<ArgJsDocArgs> = ({ tags }) => {
               <tr key={x.name}>
                 <td>
                   <code>{x.name}</code>
+                  {hasDisplayableDeprecated && (
+                    <div>
+                      <span>Deprecated</span> {tags.deprecated}
```

Comment:
given the existing implementation, shouldn't this access `tags.deprecated.name` and not `tags.deprecated`? That's what I gather from console logging a bunch of places at least.